flax/linen/module.py

```diff
@@ -22,23 +22,47 @@
 import threading
 import typing
 import weakref
-from typing import (Any, Callable, Dict, Iterable, List, NamedTuple, Mapping,
-                    Optional, Set, Tuple, Type, TypeVar, Union, overload)
+from typing import (
```

Comment:
These imports become very long, maybe we want to revert this?

Reply:
black (and thus pyink) are mostly non-configurable - the point is to adopt an autoformatter and stop worrying about this ever again - ie this is not about doing a one-time change, it's about adding an autoformatter and accepting what it does from now on.